app/encoding/encoding.go

```diff
@@ -42,6 +42,7 @@ func MakeConfig(moduleRegisters ...ModuleRegister) Config {
 	txConfig := tx.NewTxConfig(protoCodec, tx.DefaultSignModes)
 	txDecoder := txConfig.TxDecoder()
 	txDecoder = indexWrapperDecoder(txDecoder)
+	txDecoder = blobTxDecoder(txDecoder)
```

Comment:
Not sure why we didn't have this before. I needed it to print out the tx types in my test. Can extract to a separate PR if reviewers prefer.